employee.rb

```diff
@@ -0,0 +1,23 @@
+class Employee
+  def initialize(name:, salary:)
+    @name = name
+    @salary = salary
+    @reviews = []
+  end
+
+  def name
+    @name
+  end
+
+  def salary
+    @salary
+  end
+
+  def add_employee_review(review)
+    @review = review
+  end
+
+
+end
+
+suzy = Employee.new(name: "suzy", salary: 80000)
```

Comment:
This line doesn't really belong in a class file, as it's code that gets executed when the file loads.  You can just delete it.  The code that uses this class (like tests or any wrapper code that you write) will call Employee.new as it needs employees.